chromium_src/components/variations/variations_url_constants.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "components/variations/variations_url_constants.h"
+
+namespace variations {
+
+// Default server of Variations seed info.
+const char kDefaultServerUrl[] =
+    "https://variations.brave.com/seed";
+
+const char kDefaultInsecureServerUrl[] =
+    "http://variations.brave.com/seed";
```

Comment:
we probably should just not provide this and avoid using insecure connections at all (cc @diracdeltas )

Reply:
apparently the HTTP url was added as a fallback in https://bugs.chromium.org/p/chromium/issues/detail?id=769154. my best guess for why they did this is in case the client had a field trial enabled which accidentally broke HTTPS connection to the field trial server.

i don't think we need this feature. we should disable the fallback or set it to the HTTPS url.

Reply:
confirmed with folks at Chromium that we don't really need a fallback

Reply:
sounds good, will set it to use the HTTPS url in both cases

Reply:
@diracdeltas the seed file is now being signed with https://github.com/brave/brave-variations/blob/main/crypto.go. The signature will be added to the response header by fastly.